eq-author/src/App/qcodes/QcodesPage.js

```diff
@@ -30,9 +32,25 @@ const StyledMainCanvas = styled(MainCanvas)`
   max-width: 80em;
 `;
 
+const Padding = styled.div`
+  padding: 1em;
```

Comment:
const Padding = styled.div`
    margin: 2em auto 1em;
    width: 100%;
    padding: 0 0.5em 0 1em;
    max-width: 80em;
';

changes padding/margin to the same as the qcodes table when fullscreen